storage/aptosdb/src/lib.rs

```diff
@@ -1377,6 +1377,11 @@ impl DbReader for AptosDB {
                 .ok_or_else(|| AptosDbError::NotFound(format!("TableInfo for {:?}", handle)).into())
         })
     }
+
+    /// Returns whether the indexer DB has been enabled or not
+    fn indexer_enabled(&self) -> bool {
+        self.indexer.is_some()
+    }
```

Comment:
Indexer DB?  what's that?

Reply:
context: https://github.com/aptos-labs/aptos-core/pull/2191
It's a special additional on-chain index, specifically for being able to decode table keys/values (this is the last missing piece of data that we otherwise could not display, afaik). We re-use the API converter + types to avoid doubling the work; regular fullnodes will not be affected, but any of our indexer fullnodes will have this flag set for storage :-)

Reply:
So maybe put this in a different way.  Why not always enable it?

Reply:
we talked offline, but tldr; it's expensive and requires being able to stream from genesis